skyplane/replicate/replicator_client.py

```diff
@@ -576,8 +576,6 @@ def monitor_transfer(
                         # check for errors and exit if there are any (while setting debug flags)
                         errors = self.check_error_logs()
                         if any(errors.values()):
-                            copy_gateway_logs = True
```

Comment:
Why do we not copy errors upon an error? I'm OK with commenting out write_profile but write_gateway_logs is useful

Reply:
We can keep it, I think this is auto-removed when I run `autoflake --in-place --remove-all-unused-imports --remove-unused-variables --recursive skyplane`?